### ISSUE TITLE
Lyrebird won't open.

### ISSUE THREAD

**Issue by Banana1059** [lyrebird-voice-changer/lyrebird#83]
When I try to open the app it never loads.
![Screenshot 2021-08-06 8 09 38 PM](https://user-images.githubusercontent.com/88565974/128583264-52ddc7cc-d097-4567-8c16-e615db11772c.png)

**Comment by Caike555** [comment]
same

**Comment by return215** [comment]
Same here on Arch. When I try to load the app via terminal, it says this:

```
Traceback (most recent call last):
  File "/usr/share/lyrebird/app.py", line 13, in <module>
    win = mainwindow.MainWindow()
  File "/usr/share/lyrebird/app/mainwindow.py", line 62, in __init__
    utils.unload_pa_modules(check_state=False)
  File "/usr/share/lyrebird/app/core/utils.py", line 60, in unload_pa_modules
    subprocess.call('pacmd unload-module module-null-sink'.split(" "))
  File "/usr/lib/python3.9/subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'pacmd'
```

But it's probably just me since I am using Pipewire. I might have to open up a new issue instead.

**Comment by metamini** [comment]
same here

**Comment by harrego** [comment]
This is Pipewire related. I'm working on a compatibility fix, in the meantime please look at #62.
